build.gradle.kts

```diff
@@ -27,7 +27,7 @@ dependencies {
 indra {
   javaVersions {
     target(8)
-    testWith(8, 11, 16)
+    testWith(8, 11, 17)
```

Comment:
If we're updating this probably want to update build.yml as well

Reply:
Should `actions/setup-java` also be updated to `v3` to match the other repositories or should that be a separate PR that changes the workflow to <https://github.com/KyoriPowered/.github/blob/trunk/.github/workflows/shared-ci.yaml>

Reply:
Separate PR is better